operator/v1/types_ingress.go

```diff
@@ -443,7 +443,7 @@ const (
 // +union
 type ProviderLoadBalancerParameters struct {
 	// type is the underlying infrastructure provider for the load balancer.
-	// Allowed values are "AWS", "Azure", "BareMetal", "GCP", "Nutanix",
+	// Allowed values are "AWS", "Azure", "BareMetal", "GCP", "IBM", "Nutanix",
```

Comment:
Where is the change to the discriminator enum to allow this? Can't see a change to `LoadBalancerProviderType` included here

Reply:
It was already changed here: https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L473-#L490. Comment was not up to date.